include/alpaka/exec/ExecGpuCudaRt.hpp

```diff
@@ -114,8 +114,6 @@ namespace alpaka
 #if (!__GLIBCXX__) // libstdc++ even for gcc-4.9 does not support std::is_trivially_copyable.
             static_assert(
                 meta::Conjunction<
-                    // This true_ is required for the zero argument case because and_ requires at least two arguments.
```

Comment:
Could you please move this change to a separate pull request. It removes a argument from a class and we should have a documentaion why we now allowed to remove it.